### ISSUE TITLE
Pretrained model results

### ISSUE THREAD

**Issue by t-martyniuk** [jokester-zzz/SSA-SC#4]
Hello! 
Thank you for providing the code and the model checkpoint.

I used the pretrained model that you provided, and indeed, as already highlighed in #3 , there were some changes to be made. 

I assume the line `elif selected_model == 'SSA_SC':` had to be changed to `elif selected_model == 'BEV_3D_UNet':` here: https://github.com/jokester-zzz/SSA-SC/blob/ed5298dabf0ab9b1675031b04e62080664e673ad/networks/common/model.py#L49

And I also had to make some adjustments as I was using a newer version of `spconv` and other dependencies (I can detail more if needed).

I re-evaluated the model on the validation set, and achieved `IoU=57.90` and `mIoU=21.38`, which is a bit lower than in the paper (Table 2).
Do you think it could be the case that the uploaded model is not the final version that you used for the test submission / tables in the paper? 

Thank you, any hints would help.

**Comment by jokester-zzz** [comment]
Thank you so much for your interest in this work!

Regarding the checkpoint issue, my recollection is that it is indeed the model weight used in the paper. The difference in name is due to a standardization adjustment I made to the model names in the code after the experiments, resulting in a discrepancy with the previously used training weight names. I apologize for any confusion this may have caused.

You can refer to the evaluation log file `stdout.txt` provided in the weights folder. As for the issue of reproducible results, many factors can cause deviations, such as minor updates to the dataset or upgrades to dependent libraries.

I apologize, but since I haven't been following the SSC field for some time, it is difficult for me to conduct a more in-depth investigation into this specific issue.

Thank you for your understanding!

**Comment by t-martyniuk** [comment]
thank you.
